services/QuillLMS/client/app/bundles/Shared/components/shared/sortableList.tsx

```diff
@@ -29,246 +29,216 @@ const dropAnimationConfig: DropAnimation = {
   sideEffects: defaultDropAnimationSideEffects({
     styles: {
       active: {
-        opacity: "0.4"
-      }
-    }
-  })
+        opacity: '0.4',
+      },
+    },
+  }),
 };
 
 /**
  * Drag overlay to mirror the active (dragged) item.
  */
-const SortableOverlay = ({ children }) => {
-  return (
-    <DragOverlay dropAnimation={dropAnimationConfig}>
-      {children}
-    </DragOverlay>
-  );
-};
+const SortableOverlay = ({ children }) => (
+  <DragOverlay dropAnimation={dropAnimationConfig}>{children}</DragOverlay>
+);
+
+/**
+ * Custom keyboard sensor that respects certain element types and uses Enter or Space to activate.
+ */
+export class KeyboardSensorThatHandlesElementType extends KeyboardSensor {
+  static activators = [
+    {
+      eventName: 'onKeyDown',
+      handler: ({ nativeEvent: event }: React.KeyboardEvent<Element>) => {
+        if (!shouldHandleEvent(event)) {
+          return false;
+        }
+        return event.key === 'Enter' || event.key === ' ';
+      },
+    },
+  ];
+}
+
+/**
+ * Custom pointer sensor that respects certain element types.
+ */
+export class PointerSensorThatHandlesElementType extends PointerSensor {
+  static activators = [
+    {
+      eventName: 'onPointerDown',
+      handler: ({ nativeEvent: event }: React.PointerEvent<Element>) =>
+        shouldHandleEvent(event),
+    },
+  ];
+}
+
+/**
+ * Helper function to determine whether the event should be handled.
+ */
+function shouldHandleEvent(event: React.SyntheticEvent) {
+  if ((event as React.KeyboardEvent).key === 'Tab') {
+    return false;
+  }
+  const activeElement = document.activeElement;
+  if (
+    activeElement &&
+    ([INPUT, SELECT, TEXTAREA, BUTTON].includes(activeElement.tagName) ||
+      activeElement.getAttribute('contenteditable') === 'true')
+  ) {
+    return false;
+  }
```

Comment:
I don't know if this is actually any better, but another approach here to simplify your conditional would be to split the presence of `activeElement` out into its own early return:
```javascript
const activeElement = document.activeElement;

if (!activeElement) return true;

if ([INPUT, SELECT, TEXTAREA, BUTTON].includes(activeElement.tagName) || activeElement.getAttribute('contenteditable') === 'true') return false;

return true;
```